docker-compose.yml

```
@@ -1,6 +1,36 @@
 version: '3.7'
 services:
+  django:
+    image: caim-django:latest
+    build: .
+    restart: always
+    environment:
+      - PRODUCTION=0
+      - DEBUG=1
+      - MEDIA_USE_S3=0
+      - SECRET_KEY='dfdfeferewrewrewrrerewr'
+      - DB_HOST=postgres
+      - DB_PORT=5432
+      - DB_USER=postgres
+      - DB_PASSWORD=postgres
+      - DB_NAME=postgres
+      - IMAGE_RESIZE_USE_IMAGKIT=0
+      - EMAIL_ENABLED=0
+      - URL_PREFIX='http://127.0.0.1:8000'
+      - SALESFORCE_ENABLED=0
+      - SALESFORCE_USERNAME='<username>'
+      - SALESFORCE_PASSWORD='<pwd>'
+      - SALESFORCE_SECURITY_TOKEN='<token>'
+    volumes:
+      - ./seed_data:/app/seed_data
```

```suggestion
      - .:/app
```

this makes hot reloading work inside the container